Traceback (most recent call last):
  File "/home/danbot/Documents/UBC/CIVL_418_2022/Engineering_Hydrology_Notebooks/env/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/home/danbot/Documents/UBC/CIVL_418_2022/Engineering_Hydrology_Notebooks/env/lib/python3.8/site-packages/nbclient/client.py", line 1269, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/danbot/Documents/UBC/CIVL_418_2022/Engineering_Hydrology_Notebooks/env/lib/python3.8/site-packages/nbclient/util.py", line 85, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/home/danbot/Documents/UBC/CIVL_418_2022/Engineering_Hydrology_Notebooks/env/lib/python3.8/site-packages/nbclient/util.py", line 60, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/home/danbot/Documents/UBC/CIVL_418_2022/Engineering_Hydrology_Notebooks/env/lib/python3.8/site-packages/nbclient/client.py", line 707, in async_execute
    await self.async_execute_cell(
  File "/home/danbot/Documents/UBC/CIVL_418_2022/Engineering_Hydrology_Notebooks/env/lib/python3.8/site-packages/nbclient/client.py", line 1025, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/danbot/Documents/UBC/CIVL_418_2022/Engineering_Hydrology_Notebooks/env/lib/python3.8/site-packages/nbclient/client.py", line 919, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# import libraries
# --> libraries are collections of functions developed by communities of 
# open source software developers that typically focus on solving very specific problems

# pandas is a library that manipulates "dataframes", table-like structures,
# to efficiently run operations on large datasets
import pandas as pd

# numpy is a fundamental library for scientific computing
# based on the representation of data as arrays
import numpy as np

# matplotlib is a plotting library
from matplotlib import pyplot as plt
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Input [0;32mIn [1][0m, in [0;36m<cell line: 7>[0;34m()[0m
[1;32m      1[0m [38;5;66;03m# import libraries[39;00m
[1;32m      2[0m [38;5;66;03m# --> libraries are collections of functions developed by communities of [39;00m
[1;32m      3[0m [38;5;66;03m# open source software developers that typically focus on solving very specific problems[39;00m
[1;32m      4[0m 
[1;32m      5[0m [38;5;66;03m# pandas is a library that manipulates "dataframes", table-like structures,[39;00m
[1;32m      6[0m [38;5;66;03m# to efficiently run operations on large datasets[39;00m
[0;32m----> 7[0m [38;5;28;01mimport[39;00m [38;5;21;01mpandas[39;00m [38;5;28;01mas[39;00m [38;5;21;01mpd[39;00m
[1;32m      9[0m [38;5;66;03m# numpy is a fundamental library for scientific computing[39;00m
[1;32m     10[0m [38;5;66;03m# based on the representation of data as arrays[39;00m
[1;32m     11[0m [38;5;28;01mimport[39;00m [38;5;21;01mnumpy[39;00m [38;5;28;01mas[39;00m [38;5;21;01mnp[39;00m

[0;31mModuleNotFoundError[0m: No module named 'pandas'
ModuleNotFoundError: No module named 'pandas'

